src/modules/75-cf/strings/strings.en.yaml

```
@@ -288,6 +288,8 @@ staleFlagAction:
   flagsSelected: '{{ count }} Flags Selected'
   checkStaleFlag: Check potentially stale flag
   checkAllStaleFlags: Check all potentially stale flags
+  flagLifecycleExplained: Flags lifecycle explained
+  flagLifecycleDesc: Flag cleanup automation allows users to run a pipeline step that will automatically cleanup unwanted flags and any associated code within their codebase.  Think Github's dependabot but for removing flags instead.
```

HTML will collapse the extra space anyway, but worth being correct.
```suggestion
  flagLifecycleDesc: Flag cleanup automation allows users to run a pipeline step that will automatically cleanup unwanted flags and any associated code within their codebase. Think Github's dependabot but for removing flags instead.
```

---

Good spot, have fixed